source/Spatch/tests/home.test.js

```diff
@@ -0,0 +1,4 @@
+// const randomTest = require('./scripts/home.js')
+test( 'test', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group18/cse110-fa21-group18/pullRequest?prid=8453116)
```suggestion
test( "test", () => {
```